alpha/lib/model/entry.php

```diff
@@ -760,15 +760,24 @@ public function getDataUrl( $version = NULL )
 		}
 		else if ($media_type == self::ENTRY_MEDIA_TYPE_IMAGE  )
 		{
-			$width = self::DEFAULT_IMAGE_WIDTH;
-			$height = self::DEFAULT_IMAGE_HEIGHT;
+			$dataKey = $this->getSyncKey(self::ENTRY_MEDIA_SOURCE_FILE);
+			$file_sync = kFileSyncUtils::getReadyExternalFileSyncForKey($dataKey);
+			if($file_sync)
+			{
+				$url =  $file_sync->getExternalUrl($this->getId());
```

Comment:
getDataUrl() should try to return the actual data file not thumbnail just like videos. If can't get the file then fall back to the thumbnail